lambdas/functions/management/management.go

```diff
@@ -0,0 +1,78 @@
+package management
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/weka/go-cloud-lib/connectors"
+	"github.com/weka/go-cloud-lib/lib/jrpc"
+	"github.com/weka/go-cloud-lib/lib/weka"
+	"github.com/weka/go-cloud-lib/logging"
+	"github.com/weka/go-cloud-lib/protocol"
+)
+
+type WekaRequest struct {
+	Username          string   `json:"username"`
+	Password          string   `json:"password"`
+	BackendPrivateIps []string `json:"backend_private_ips"`
+}
+
+// ManagementRequest is the request object for the management lambda
+// The operation to be performed is specified in the Type field.  The specific
+// operation payload is in the WekaRequest field. The convention here is that
+// each type value should have a corresponding embedded struct that contains
+// the payload for that specific operation.
+//
+// At this time, the only operation supported is "status"
+type ManagementRequest struct {
```

Comment:
this struct can be removed, we should only use `WekaApiRequest` and do the required updates to the current status implementation